modules/ROOT/pages/cloudhub-impaired-worker.adoc

```diff
@@ -1,17 +1,59 @@
+CloudHub continuously performs health checks on the underlying virtual machines where Mule applications are running. 
+When a worker is impaired, CloudHub automatically attempts to restart the worker and relaunch applications.
 
-[NOTE]
+A worker is impaired when there is a hardware or software failure that negatively impacts the running application deployed on the worker.
+Hardware and software failures include network unavailability, memory issues, server hardware failures, and file corruption.
+
+When CloudHub restarts the impaired worker:
+
+* If a retry is successful, the underlying worker and app restart without any further action.
++
 It might take up to five minutes for a worker to restart.
+* If the application fails to start after five attempts, the status of the application is *Failed* or *Unrecoverable Runtime Error*.
++
+In this case, you must manually restart the application.
+
+The logs indicate when a worker is restarted automatically and any failures.
 
-If the application fails to start after five attempts, the status of the application is *Failed* or *Unrecoverable Runtime Error*. In this case, you must manually redeploy and restart the application.
+[NOTE]
+The automatic worker-restart functionality is different from the automatic application-restart functionality, which is configurable using the *Automatically restart application when not responding* option on the *Settings* page for the app. 
+CloudHub automatically attempts to restart impaired workers, regardless of the app setting. 
 
 == Scheduled Maintenance Activities
 
-CloudHub checks for scheduled maintenance activities on the worker instances running the Mule applications. The following maintenance activities can cause a potential downtime for the running applications: reboot, stop, start, or retirement of the instances. To avoid downtime, Cloudhub checks for these scheduled events and sends you a notification in advance to restart your application with zero downtime.
+CloudHub checks for scheduled maintenance activities for worker instances that cause potential downtime for the running applications: reboot, stop, start, or retirement of the instances.
+
+A notification appears in the top right of both the *Applications* page and the *Settings* page for the application, notifying you of upcoming maintenance activities:
+
+.The arrow shows the notification icon on the *Settings* page for the application.
+image::cloudhub-notification-icon.png[Notification icon on the *Settings* page for the application]
+
+Click the icon to display the notification, which includes the date that CloudHub will automatically restart the worker.
+
+
+You can restart your apps manually or wait for CloudHub to restart them automatically. A reminder notification appears one week before the automatic application restart.
+
+If CloudHub restarts the app, the following message appears in the application log:
```

Comment:
Does CH restart the app, or does the app automatically restart because of how it is configured? 
Is this scenario (maint) different from failure?

Reply:
CH restarts the app. This is a restart of the worker, which is different than the auto-app restart. The app is restarted as a result of the worker restart.